tests/integration/test_cli_dedupe_removal_copilot.py

```
@@ -0,0 +1,409 @@
+    """Tests for remove_files()."""
+
+    def test_dry_run_does_not_delete(self, tmp_path: Path) -> None:
+        from rich.console import Console
+
+        from file_organizer.cli.dedupe_removal import remove_files
+
+        f = tmp_path / "duplicate.txt"
+        f.write_text("data")
+
+        files = [_make_file_dict(f, size=100)]
+        console = Console(file=open("/dev/null", "w"), no_color=True)
```

Ruff (0.15.9)

[warning] Use a context manager for opening files

(SIM115)